lib/install.ps1

```diff
@@ -34,9 +34,14 @@ function install_app($app, $architecture, $global, $suggested, $use_cache = $tru
         return
     }
 
-    if ((get_config 'manifest-review' $false) -and ($MyInvocation.ScriptName -notlike '*scoop-update*')) {
-        Write-Output "Manifest: $app.json"
-        Write-Output $manifest | ConvertToPrettyJson
+    if ((get_config 'manifest_review' $false) -and ($MyInvocation.ScriptName -notlike '*scoop-update*')) {
+        "Manifest: $app.json"
```

Comment:
```suggestion
        Write-Host "Manifest: $app.json"
```

Do we need this line for pipeline? If not, use `Write-Host`

Reply:
Sure

Reply:
See edited comment.

Reply:
Write-Host is better, since we don't need to pass it to pipeline

Reply:
Hmm, I thought you'll use this line... 🤣 

Then `Write-Host`